assets/src/settings-page/analytics.js

```diff
@@ -125,18 +129,15 @@ function AnalyticsEntry( { entryIndex, onChange, onDelete, type = '', config = '
 				}
 			</h4>
 			<div className="amp-analytics-entry__options" id={ `amp-analytics-entry-${ String( entryIndex ) }` }>
-				<div className="amp-analytics-entry__text-inputs">
-					<TextControl
-						className="option-input"
+				<div className="amp-analytics-entry__select-control">
+					<SelectControl
+						className="option-select"
 						label={ __( 'Type:', 'amp' ) }
 						onChange={ ( newType ) => {
 							onChange( { type: newType } );
 						} }
-						type="text"
-						pattern="^[a-zA-Z0-9_.-]+$"
-						title={ __( 'Must be a valid vendor or left blank for in-house analytics.', 'amp' ) }
```

Comment:
It seems the new dropdown doesn't allow for the an empty vendor to be supplied? The ability to specific an empty vendor is needed still for in-house analytics.

Reply:
There is confusion regarding this also discussed [here](https://github.com/ampproject/amp-wp/pull/6894#discussion_r802854794) 
both `Google Tag Manager` and in-house analytics expect a `''` empty string which when adding to the dropdown conflicts in value and the first option with empty value gets selected.

What do you suggest for resolving this?

Reply:
@westonruter @delawski I need your suggestions for the above comment.

Reply:
@rutviksavsani That's a good question. I think we have two possibilities here:

- Use an empty option labelled like "None" or "---" as the first item on the list and provide a help note that for Google Tag Manager an empty vendor type should be selected.
- Instead of "None" or "---", use something like: "In house analytics / Google Tag Manager" for the empty option label.

@westonruter What are your thoughts?

Reply:
@delawski 
The second suggestion seems a more approachable way to resolve the confusion and also informative for the user at the same time without using any extra elements.
@westonruter what do you say?